class-wc-facebookcommerce.php

```diff
@@ -127,7 +127,6 @@ public function __construct() {
 		 */
 		public function init() {
 
-			add_action( 'init', array( $this, 'get_integration' ) );
```

Comment:
Early Facebook Integration object initialisation when Facebook access token is not here present. Removing it, since it is not ever necessary.